src/util/pelz_json_parser.c

```diff
@@ -547,4 +546,10 @@ int decrypt_parser(cJSON * json, charbuf * key_id, charbuf * data, charbuf * req
   }
   ///
   return (0);
+}
+
+int validate_signature(charbuf * request_sig, charbuf * requestor_cert)
```

Comment:
I mentioned this above, but this function will need both the signature and the data over which that signature was computed (in addition to the public key in the requestor cert) in order to validate the signature over the request.